modules/integration_azure-functions/conf/readme.yaml

```diff
@@ -1,3 +1,7 @@
 documentations:
   - name: Azure Monitor metrics
     url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
+  - name: SignalFx Azure Functions
+    url: 'https://docs.signalfx.com/en/latest/integrations/azure-functions.html'
+notes: |
+  If you want to enable azure.function.errors detector, you will need to instrusment your function. The wrapper for C#, Java and NodeJs is provided by SignalFX. But for Python, you will have to use [signalfx-azure-function-python](https://github.com/claranet/signalfx-azure-function-python)
```

Comment:
may be can you use `source_doc` to make this even more visible like:

```
source_doc: |
  ### Serverless wrapper

  The detector xx is based on signalfx serverless wrapper lib which allow .. available on ..
```

Reply:
Applied.